llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -4442,6 +4442,7 @@ static bool willGenerateVectors(VPlan &Plan, ElementCount VF,
       case VPDef::VPInstructionSC:
       case VPDef::VPCanonicalIVPHISC:
       case VPDef::VPVectorPointerSC:
+      case VPDef::VPReverseVectorPointerSC:
```

Comment:
This should be folded into https://github.com/llvm/llvm-project/pull/110974, as otherwise it may change behavior?

Reply:
Yes, it is gone after rebase.